backend/wmg/api/v1.py

```diff
@@ -165,6 +165,7 @@ def build_filter_dims_values(criteria: WmgFiltersQueryCriteria, snapshot: WmgSna
         "development_stage_ontology_term_id": "",
         "self_reported_ethnicity_ontology_term_id": "",
         "tissue_ontology_term_id": "",
+        "cell_type_ontology_term_id": "",
```

Comment:
This is a necessary (and otherwise harmless) change for generating the Source Data table in the cell guide data pipeline. We are including it here preemptively for convenience.